README.md

```diff
@@ -1,6 +1,34 @@
-# osdfir-infrastructure
-This repository contains a set of Helm charts and deployment scripts that can be used to deploy Open Source Digital Forensics & Incident Response tools to a Kubernetes cluster. 
+# OSDFIR Infrastructure
+OSDFIR Infrastructure helps setup Open Source
+Digital Forensics tools to Kubernetes clusters using Helm. 
 
-This is still early stages of development and not usable yet, but please stay tuned for instructions on how to install our official release. 
+Currently, OSDFIR Infrastructure
+supports the deployment of the following tools:
+  * Timesketch; ref https://github.com/google/timesketch
+  * Turbinia; ref https://github.com/google/turbinia
 
-We look forward to sharing what is in store with the community soon!
+## TL;DR
+
+```console
+helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
```

Comment:
What does `my-release` mean here? I opt to make it clear what to use here and what it means. e.g. Is there another release I can deploy?

Reply:
Yeah for example you can have a `prod` and `dev` release. I was hoping to note that this can be anything really depending on what type of deployment. If you for example ran helm install prod <timesketch-chart> and helm install dev <timesketch-chart> you will have two timesketch instances running with different release names/pods. I will add a note to make this a little more clear